t/job-manager/list-jobs.c

```diff
@@ -88,39 +73,9 @@ int main (int argc, char *argv[])
         log_err_exit ("flux_rpc_get_unpack");
 
     json_array_foreach (jobs, index, value) {
-        flux_jobid_t id;
-        int urgency;
-        unsigned int priority;
-        uint32_t userid;
-        double t_submit;
-        char timestr[80];
-        flux_job_state_t state;
-        json_t *annotations = NULL;
-        char *annotations_str = NULL;
-
-        if (json_unpack (value, "{s:I s:i s:i s:i s:f s:i s?:o}",
-                                "id", &id,
-                                "urgency", &urgency,
-                                "priority", &priority,
-                                "userid", &userid,
-                                "t_submit", &t_submit,
-                                "state", &state,
-                                "annotations", &annotations) < 0)
-            log_msg_exit ("error parsing job data");
-        if (iso_timestr (t_submit, timestr, sizeof (timestr)) < 0)
-            log_err_exit ("time conversion error");
-        if (annotations)
-            annotations_str = json_dumps (annotations, 0);
-        printf ("%ju\t%s\t%lu\t%d\t%u\t%s%s%s\n",
-                (uintmax_t)id,
-                flux_job_statetostr (state, true),
-                (unsigned long)userid,
-                urgency,
-                priority,
-                timestr,
-                annotations_str ? "\t" : "",
-                annotations_str ? annotations_str : "");
-        free (annotations_str);
+        char *s = json_dumps (value, JSON_COMPACT);
```

Comment:
replace 3 lines with `json_dumpf()`?

Reply:
ah, didn't know about `dumpf()`!

Reply:
doh, `json_dumpf()` doesn't add the newline necessary for each entry.  So gotta add an extra `printf("\n");`, well, we saved 1 lline :-)